prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -683,15 +683,11 @@ public async Task<ActionResult> GetSiteAdjudicationDocument(int siteId, int docu
         public async Task<ActionResult> PecAssignable(int siteId, string pec)
         {
             var site = await _communitySiteService.GetSiteAsync(siteId);
-            if (site == null)
```

Comment:
What is the reason to remove this check, Since if site==null here this endpoint returns nothing.

Reply:
It was not being used

Reply:
Now using `SiteExists()`